stubs/pywin32/win32/win32gui.pyi

```diff
@@ -26,6 +26,12 @@ def GetWindowLong(hwnd: int, index): ...
 def GetClassLong(hwnd: int, index): ...
 def SetWindowLong(hwnd: int, index, value): ...
 def CallWindowProc(wndproc, hwnd: int, msg, wparam, lparam): ...
+def CascadeWindows(
+    __hwndObject: _win32typing.PyHANDLE | int | None,
+    __how: int,
+    __rectObject: _win32typing.PyRECT | tuple[int, int, int, int] | int | None = None,
+    __childrenObject: tuple[_win32typing.PyHANDLE | int, ...] | None = None,
+) -> int: ...
```

Comment:
Oh I tried this function out on my machine locally to check you had the signature right and was surprised when it, you know, made my windows cascade lol

Reply:
Source for the params:
https://github.com/mhammond/pywin32/blob/bff5b3ac44596eff8a30dea139b1c63cf2487931/win32/src/win32gui.i#L3039-L3044

And yeah I tested the params to make sure the doc comments were accurate. (for example it has to be a tuple, doesn't support lists)